packages/react/src/components/Dialog/DialogFooter.tsx

```diff
@@ -0,0 +1,53 @@
+import * as React from 'react'
+import * as customPropTypes from '@stardust-ui/react-proptypes'
+
+import {
+  createShorthandFactory,
+  UIComponent,
+  UIComponentProps,
+  ChildrenComponentProps,
+  ContentComponentProps,
+  commonPropTypes,
+  ShorthandFactory,
+  childrenExist,
+} from '../../lib'
+
+import { ShorthandValue, WithAsProp, withSafeTypeForAs } from '../../types'
+import { BoxProps } from '../Box/Box'
+
+export interface DialogFooterProps
+  extends UIComponentProps,
+    ChildrenComponentProps,
+    ContentComponentProps {
+  /** Dialog actions */
+  actions: ShorthandValue<BoxProps>
+}
+
+class DialogFooter extends UIComponent<WithAsProp<DialogFooterProps>> {
+  static create: ShorthandFactory<DialogFooterProps>
+
+  static displayName = 'DialogFooter'
+  static className = 'ui-dialog__footer'
+
+  static propTypes = {
+    ...commonPropTypes.createCommon(),
+    actions: customPropTypes.itemShorthand,
+  }
+
+  renderComponent({ ElementType, classes, unhandledProps }): React.ReactNode {
+    const { children, actions, styles } = this.props
+
+    return (
+      <ElementType style={{ styles }} className={classes.root} {...unhandledProps}>
+        {childrenExist(children) ? children : actions}
```

Comment:
```suggestion
        {childrenExist(children) ? children : content}
```

I don't any reason to have there `actions`, at least as alias for `children`. For this alias we always use `content`.
FYI: @miroslavstastny